Add Usage Documentation

Hi,

This is a brand new debian install with retroarch installed via flatpak.

The first thing I tried was to update the core info files. But in my terminal it tells me:

> [ERROR] Download of 'info.zip' failed: Write failed.

A quick strace later I see:

> [pid 1829] open("/app/share/libretro/info/info.zip", O_WRONLY|O_CREAT|O_TRUNC, 0666) = -1 EROFS (Read-only file system).

Now /app doesn't exist and since the retroarch-with-flatpak documentation doesn't mention this i'm struggling to understand what's supposed to happen here. I don't expect that i'm supposed to create /app as a non root user on the system. So I wonder if some other magic is supposed to be in play here?

If I look at the retroarch.cfg file (from ~/.var/app/org.libretro.RetroArch/config/retroarch/retroarch.cfg) file I see a couple of different varieties of path:

```
assets_directory = "/app/share/libretro/assets/"
audio_filter_dir = "/app/lib/retroarch/filters/audio"
cheat_database_path = "/app/share/libretro/database/cht"
content_database_path = "/app/share/libretro/database/rdb"
content_favorites_path = "~/.var/app/org.libretro.RetroArch/config/retroarch/content_favorites.lpl"
content_history_path = "~/.var/app/org.libretro.RetroArch/config/retroarch/content_history.lpl"
content_image_history_path = "~/.var/app/org.libretro.RetroArch/config/retroarch/content_image_history.lpl"
content_music_history_path = "~/.var/app/org.libretro.RetroArch/config/retroarch/content_music_history.lpl"
content_video_history_path = "~/.var/app/org.libretro.RetroArch/config/retroarch/content_video_history.lpl"
core_assets_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/downloads"
cursor_directory = "/app/share/libretro/database/cursors"
input_remapping_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/config/remaps"
joypad_autoconfig_dir = "/app/share/libretro/autoconfig"
libretro_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/cores"
libretro_info_path = "/app/share/libretro/info"
overlay_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/overlay"
playlist_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/playlists"
rgui_config_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/config"
screenshot_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/screenshots"
thumbnails_directory = "~/.var/app/org.libretro.RetroArch/config/retroarch/thumbnails"
video_filter_dir = "/app/lib/retroarch/filters/video"
video_shader_dir = "/app/share/libretro/shaders"

```

The things starting with ~/.var/app.....  seem to be paths that exist and presumably anything referencing these work fine.

I wondered if /app in the others was supposed to be something that should have been replaced with ~/.var/app in some way (even if that were manually). But not all of the remainder of the paths starting with /app have matching (or similar) directories available under ~/.var/app.

If I take the last entry as an example /app/share/libretro/shaders... there's an almost matching ~/.var/app/org.libretro.RetroArch/config/retroarch/shaders
But if I look for something like /app/share/libretro/info, I can't find anything under ~/.var/app that's even remotely like it.

Which suggests to me that these things probably aren't expected to live under ~/.var/app at all.

I then looked in my users whole home directory and something like /app/share/libretro/assets/ seems to have something much more like a match under:
~/.local/share/flatpak/app/org.libretro.RetroArch/x86_64/stable/a437b0ef4eb1f1563d84a8ec881b9f19eb9fb84c06b35ecafe4ce85d5aad918a/files/share/libretro/assets
 
I looked at a few more and they also seem to have matching things under ~/.local/share/flatpak/app/org.libretro.RetroArch/x86_64/stable/a437b0ef4eb1f1563d84a8ec881b9f19eb9fb84c06b35ecafe4ce85d5aad918a/files

So I think I could probably fix my problem by simply replacing all references to /app in my retroarch.cfg with ~/.local/share/flatpak/app/org.libretro.RetroArch/x86_64/stable/a437b0ef4eb1f1563d84a8ec881b9f19eb9fb84c06b35ecafe4ce85d5aad918a/files

But I quickly realised that it seems silly that I should have to do that and would cause problems should ~/.local/share/flatpak/app/org.libretro.RetroArch/x86_64/stable/a437b0ef4eb1f1563d84a8ec881b9f19eb9fb84c06b35ecafe4ce85d5aad918a/files ever change.

Is there something with flatpak that i'm missing that should be doing this automatic translation of /app pathnames for me in some way? 

I've had a hunt around trying to work out if /app has some special meaning to flatpak, but so far have not managed to find anything useful.

Thanks

--- a/README.md
+++ b/README.md
@@ -26,7 +26,7 @@ flatpak install --user -y flathub org.libretro.RetroArch
 
 4. [Launch content](https://docs.libretro.com/guides/launch-content/) through RetroArch either through the menu, or through the command line:
     ```
-    flatpak run org.libretro.RetroArch -L ~/.var/app/org.libretro.RetroArch/data/cores/chailove_libretro.so FloppyBird.chailove
+    flatpak run org.libretro.RetroArch -L ~/.var/app/org.libretro.RetroArch/config/retroarch/cores/chailove_libretro.so FloppyBird.chailove
     ```
 
 ## Update

--- a/README.md
+++ b/README.md
@@ -13,12 +13,21 @@ flatpak remote-add --user --if-not-exists flathub https://flathub.org/repo/flath
 flatpak install --user -y flathub org.libretro.RetroArch
 ```
 
-## Run
+## Usage
 
-To run RetroArch through Flatpak, execute:
-```
-flatpak run org.libretro.RetroArch
-```
+1. Run RetroArch through Flatpak:
+    ```
+    flatpak run org.libretro.RetroArch
+    ```
+
+2. [Install some libretro cores](https://docs.libretro.com/guides/download-cores/) using the Online Updater. There is no need to update core info files, assets, joypad profiles, cheats, database, cg, glsl, or slang shaders, as those are shipped with the Flatpak.
+
+3. [Import content](https://docs.libretro.com/guides/import-content/) by scanning the folder where your games are kept.
+
+4. [Launch content](https://docs.libretro.com/guides/launch-content/) through RetroArch either through the menu, or through the command line:
+    ```
+    flatpak run org.libretro.RetroArch -L ~/.var/app/org.libretro.RetroArch/data/cores/chailove_libretro.so FloppyBird.chailove
+    ```
 
 ## Update
 

